keras_hub/src/models/segformer/segformer_presets.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2024 The KerasHub Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""SegFormer model preset configurations."""
+
+presets_no_weights = {
```

Comment:
These are based on MiTB0...MiTB5 presets, which don't seem to be part of the presets on KerasHub.
Does KerasHub have an opinionated way of handling the `from_preset()` constructors?

@divyashreepathihalli @fchollet

Reply:
Do we use the same alias-type [approach](https://github.com/keras-team/keras-cv/blob/master/keras_cv/src/models/segmentation/segformer/segformer_aliases.py) from before such as here?